src/history.cpp

```diff
@@ -1811,7 +1811,7 @@ wcstring history_session_id() {
     } else if (valid_var_name(session_id)) {
         result = session_id;
     } else {
-        debug(0, "Wrong characters in the session ID: '%ls'. Using `fish`.",
+        debug(0, _(L"Wrong characters in the session ID: '%ls'. Using `fish`."),
```

Comment:
Something like "History session IDs must be valid fish variable names" would be more helpful. Simply telling the user they included "wrong characters" doesn't help them correct the error.